### PR TITLE
Add missing disclosures

### PR DESCRIPTION
**Pull request by maxammann**
This table was automatically generated from our internal database.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -749,9 +749,22 @@ Companies that have allowed us to speak about our work can be found here. Many m
...
+|Name|Product|Discoverer|Date|ID|Blog|
+|---|---|---|---|---|---|
+|Numbers turned weapons: DoS in Osmosis’ math library|Osmosis|Sam Alws|2023|❌|[💬](https://blog.trailofbits.com/2023/10/23/numbers-turned-weapons-dos-in-osmosis-math-library/)|
+|Security flaws in an SSO plugin for Caddy|caddy-security|Maciej Domanski, Travis Peters, and David Pokora|2023|❌|[💬](https://blog.trailofbits.com/2023/09/18/security-flaws-in-an-sso-plugin-for-caddy/)|
+|ktor Path Traversal|ktor|Vasco Franco|2023|CVE-2022-48476|❔|
+|Specialized Zero-Knowledge Proof failures|Binance's tss-lib; All forks of tss-lib: Joltify, SwipeChain, and ThorChain; Coinbase's kryptology|Opal Wright|2022|❌|[💬](https://blog.trailofbits.com/2022/11/29/specialized-zero-knowledge-proof-failures/)|
+|Forgery in Amis' Alice library|[Amis' alice](https://github.com/getamis/alice#acknowledgments)|Filipe Casal|2022|❌|❔|
+|The issue with ATS in Apple’s macOS and iOS|macOS and iOS|Will Brattain|2022|CVE-2023-38596|[💬](https://blog.trailofbits.com/2023/10/30/the-issue-with-ats-in-apples-macos-and-ios/)|
```
Comment (traviswpeters) [inline README.md:759]: We just updated this in the post earlier this week. This one impacted iOS, iPadOS, tvOS, macOS, and watchOS.

File: README.md [hunk 2]
```diff
@@ -749,9 +749,22 @@ Companies that have allowed us to speak about our work can be found here. Many m
...
+|Name|Product|Discoverer|Date|ID|Blog|
+|---|---|---|---|---|---|
+|Numbers turned weapons: DoS in Osmosis’ math library|Osmosis|Sam Alws|2023|❌|[💬](https://blog.trailofbits.com/2023/10/23/numbers-turned-weapons-dos-in-osmosis-math-library/)|
+|Security flaws in an SSO plugin for Caddy|caddy-security|Maciej Domanski, Travis Peters, and David Pokora|2023|❌|[💬](https://blog.trailofbits.com/2023/09/18/security-flaws-in-an-sso-plugin-for-caddy/)|
+|ktor Path Traversal|ktor|Vasco Franco|2023|CVE-2022-48476|❔|
+|Specialized Zero-Knowledge Proof failures|Binance's tss-lib; All forks of tss-lib: Joltify, SwipeChain, and ThorChain; Coinbase's kryptology|Opal Wright|2022|❌|[💬](https://blog.trailofbits.com/2022/11/29/specialized-zero-knowledge-proof-failures/)|
+|Forgery in Amis' Alice library|[Amis' alice](https://github.com/getamis/alice#acknowledgments)|Filipe Casal|2022|❌|❔|
```
Comment (traviswpeters) [inline README.md:758]: For entries with no blog post, I recommend leaving it blank rather than putting a question mark.

**Comment by maxammann** [comment]
I'll come back to this once the missing blockchain disclosures are added.

File: README.md (hunk 1 above)
Comment (maxammann) [reply README.md]: adressed

File: README.md (hunk 2 above)
Comment (maxammann) [reply README.md]: adressed

**Comment by maxammann** [comment]
@dykstra10 this is ready for your review

**Review by dykstra10 — APPROVED** [review]

**Pull request merged**